js/help.js

```diff
@@ -0,0 +1,49 @@
+const helpOverlay = new Image();
+helpOverlay.src = 'images/help-image.png';
+
+const helpBtn = document.querySelector('.help');
+
+let helpImage = false;
+
+function drawHelp() {
+  if (helpImage === true) {
+    ctx.drawImage(helpOverlay, 15, 0);
+  }
+}
+
+function introText() {
+  //Intro text
+  ctx.clearRect(0, 0, canvasWidth, canvasHeight);
+  ctx.font = '3rem "Carter One"';
+  ctx.fillStyle = 'gainsboro';
+  ctx.textBaseline = 'middle';
+  ctx.textAlign = 'center';
+  ctx.fillText(
+    `Start a new game to tile a wall!`,
```

Comment:
Better not to hardcode numbers and strings.
In graphics hardcoding numbers is kind of unavoidable, but, regarding texts: it's better to keep the object with them separately.